security/firely-auth/firely-auth-settings.rst

```diff
@@ -390,6 +416,10 @@ External identity providers
 -	``AllowAutoProvision``: true / false - If true, Firely Auth will automatically create a user in its own database if the user logs in with an external identity provider for the first time. The user will be created with the claims that are provided by the external identity provider.
 - ``AutoProvisionFromSecurityGroup``: When ``AllowAutoProvision`` is true, this setting allows you to specify a security group that the user must be a member of in order to be automatically provisioned. If the user is not a member of this group, the user will not be automatically provisioned.
 - ``UserClaimsFromIdToken``: This setting allows you to map the claims from the token that is received from the external identity provider to the claims that are stored in the Firely Auth database. The key is the claim that is received from the external identity provider. This key can be copied as a value that is recognized by Firely Auth. For intance, Azure is able to provide fhirUser claim to the token, but will prefix this claim with ``extn.``. The CopyAs field can be used to remove this prefix, so that Firely Auth is able to recognize the fhirUser claim.
+- ``FhirUserLookupClaimsMapping``: As an alternative for retrieving the FhirUser Claim from the ``UserClaimsFromIdToken`` setting, ``FhirUserLookupClaimsMapping`` allows you to use the claims from the ID token to search for a users respective resource in Firely Server. This can either be a Patient resource or a Practitioner recource. Firely Auth will then use the id of this resource to derive the FhirUser claim of the user upon SSO auto-provisioning.
+- ``SearchParameterName``: The search parameter that will be used to search for the user in Firely Server. This can be any search parameter that can be used to query ``Patient`` or ``Practitioner`` resources.
```

Comment:
```suggestion
- ``SearchParameterName``: The search parameter that will be used to search for the user in Firely Server. This can be any search parameter that can be used to query ``Patient`` or ``Practitioner`` resources. This search parameter will be used on a system-level search against Firely Server.
```